configure.ac: check egrep

The macro AC_HEADER_STDC which requires AC_PROG_EGREP has been removed
from configure.ac, then it fails to substitute '@EGREP@':

|     [GEN] command-lines-input.h
| /bin/bash: line 2: @EGREP@: command not found
|     [GEN] command-count.h

Call AC_PROG_EGREP to fix the error.

Upstream-Status: Submitted [https://gitlab.com/lvmteam/lvm2/-/merge_requests/4]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 2e52498f9..d9462e22c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -78,6 +78,7 @@ AC_PROG_LN_S
 AC_PROG_MAKE_SET
 AC_PROG_MKDIR_P
 AC_PROG_RANLIB
+AC_PROG_EGREP
 AC_CHECK_TOOL([READELF], [readelf])
 AC_CHECK_TOOL(AR, ar)
 AC_PATH_TOOL(CFLOW_CMD, cflow)
-- 
2.34.1

